apps/emqx/src/emqx_app.erl

```diff
@@ -48,6 +48,7 @@
 start(_Type, _Args) ->
     ok = maybe_load_config(),
     %% Load application first for ekka_mnesia scanner
+    mnesia:change_table_copy_type(schema, node(), disc_copies),
```

Comment:
@zmstone I have to add this to make the application able to start, otherwise got 
```{"Kernel pid terminated",application_controller,"{application_start_failure,emqx,{bad_return,{{emqx_app,start,[normal,[]]},{'EXIT',{{badmatch,{bad_type,emqx_rule,disc_copies,'emqx@127.0.0.1'}},[{emqx_rule_registry,mnesia,1,[{file,\"src/emqx_rule_registry.erl\"},{line,109}]},{ekka_boot,'-apply_module_attributes/1-lc$^1/1-1-',2,[{file,\"/emqx/deps/ekka/src/ekka_boot.erl\"},{line,23}]},{ekka_boot,'-apply_module_attributes/1-lc$^0/1-0-',1,[{file,\"/emqx/deps/ekka/src/ekka_boot.erl\"},{line,23}]},{ekka_mnesia,start,0,[{file,\"/emqx/deps/ekka/src/ekka_mnesia.erl\"},{line,104}]},{ekka,start,0,[{file,\"/emqx/deps/ekka/src/ekka.erl\"},{line,96}]},{emqx_app,start,2,[{file,\"src/emqx_app.erl\"},{line,52}]},{application_master,start_it_old,4,[{file,\"application_master.erl\"},{line,293}]}]}}}}}"}
```